python/python/lancedb/query.py

```diff
@@ -1815,6 +1844,7 @@ def __init__(self, inner: LanceVectorQuery):
         """
         super().__init__(inner)
         self._inner = inner
+        self._reranker = RRFReranker()
```

Comment:
Similar to my comment on AsyncFTSQuery, by default the reranker can probably be `None`

This is similar to the synchronous python SDK:
https://github.com/lancedb/lancedb/blob/50c30c5d34dab481ca00897ed9fb63aa8b033567/python/python/lancedb/query.py#L610